systers_portal/templates/meetup/snippets/meetup_sidebar.html

```diff
@@ -0,0 +1,9 @@
+<div class="sidebar-module mb40">
```

Comment:
Try to batch common actions together. For example under Meetup Location Actions (Not **Community**) we should have "Add New Meetup" and "Add New Member". This is necessary to hide the whole block if the user is not authenticated or does not have permissions.

Then a separate div should contain the about button. Add a separate block for Join button. All those are better be stored in different snippet files. 

For reference see this page (logged in) - http://systersportal.herokuapp.com/community/systers/p/about/